server/src/main/java/keywhiz/ServiceModule.java

```diff
@@ -75,6 +87,14 @@ public ServiceModule(KeywhizConfig config, Environment environment) {
     bind(KeywhizConfig.class).toInstance(config);
   }
 
+  private String generateCookieKey() {
```

Comment:
This means the cookie key won't work across hosts, and it'll also get wiped on restart and everyone ends up being logged out?